src/rulesets/finder.ts

```diff
@@ -75,7 +76,7 @@ export async function findFile(from: string, to: string) {
 
 function exists(uri: string): Promise<boolean> {
   return new Promise<boolean>(resolve => {
-    fs.access(uri, fs.constants.F_OK, err => {
+    fs.stat(uri, err => {
```

Comment:
@P0lip Why this change? The doc seems to state _"To check if a file exists without manipulating it afterwards, `fs.access()` is recommended."_

Another approach would be to just use `existsSync`.

Thoughts?

Reply:
It's because of studio Web that does not support fs.access.

Reply:
Ok.... Although that's out of the scope of that's PR, maybe should we implement some kind of a jest global `fs` mock that fails when the core leverages functions that aren't supported by studio Web?